schemas/org.notes.gschema.xml

```diff
@@ -119,7 +119,6 @@
 			<summary>Stylesheet modifications for renderer to use</summary>
 			<description>This data is the actual CSS the renderer will use</description>
 		</key>
-	  	<child name="notebooks" schema="org.notes.notebook_data.notebook" />
```

Comment:
Is there a purpose for this line? This child element contains itself, so every-time I run 'gsettings list-recursively' (I don't like using dconf) I end up in an infinite loop.